app/controllers/concerns/lol_connections.rb

```diff
@@ -1,127 +0,0 @@
-require 'lol'
```

Comment:
I fixed the searching error in this commit: https://github.com/danfstucky/LeagueSpec/pull/47/commits/7f23c57ea9676451acaf6081ea991042fc89f9a0

There are all sorts of problems dealing with friendships and this refactor still.  I've given it a lot of investigation and I may add the `lol_connections` file back for this review and create another refactoring issue to handle it later.  The reason is because the friend invitation functionality that you are currently working on will break anything I do to fix the friendship stuff here so I think it would be better to wait until your pull request is completed and merged and then refactor all of that together in a separate issue.